lambdanotes/src/App.js

```diff
@@ -3,7 +3,7 @@ import './App.css';
 import Sidebar from "./components/sidebar/Sidebar"
 import NewNote from "./components/newnote/NewNote";
 import NoteContainer from "./components/notes/NoteContainer";
-import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { BrowserRouter as Route} from "react-router-dom";
```

Comment:
@Raymondgrc very cool on trying to use a git feature such as `revert`! Let's talk tomorrow morning to get this resolved so you can continue with the project 👍